### ISSUE TITLE
Multiple Network Factories

### ISSUE THREAD

**Issue by victormer** [ponder-sh/ponder#961]
Hey there,

I found what I think it's a bug.

I'm trying to have multiple networks supporting a factory. What I experience is:

1. When I config both networks (local + sepolia) it works fine with sepolia contracts **except for the sepolia factory**.
2. When I config **ONLY** the sepolia network, it **works fine with all contracts** (including factory).
3. When I config both networks and swap the name of the networks, it's the local factory that it's not registering assets.
4. The event of the factory is being captured for the logs I see, but no entity is being created to register events.

You can see results here:

ONLY SEPOLIA

![Screenshot 2024-07-07 at 09 53 20](https://github.com/ponder-sh/ponder/assets/38182181/0248e85f-490f-413d-9f50-252a3f9eb701)
![Screenshot 2024-07-07 at 09 53 32](https://github.com/ponder-sh/ponder/assets/38182181/850a9a2d-8e8a-4bf4-85e5-93a82303f594)

BOTH

![Screenshot 2024-07-07 at 09 52 05](https://github.com/ponder-sh/ponder/assets/38182181/4536749e-4143-4f6b-af77-c9a04a470366)
![Screenshot 2024-07-07 at 09 53 48](https://github.com/ponder-sh/ponder/assets/38182181/2c7a3117-360b-4011-9007-4c51fbfe3607)

**Comment by kyscott18** [comment]
Sorry about this. Is your foundry local network configured with interval mining? This seems like it could be causing the behavior that you are seeing.

https://ponder.sh/docs/advanced/foundry#mining-mode

**Comment by victormer** [comment]
Exactly, the error disappeared when I set an interval of 2 seconds for blocks. Thanks for the response.
